bspump/application.py

```diff
@@ -63,11 +63,21 @@ def __init__(self, args=None):
             self.WebContainer = initialize_web(self.WebService.WebContainer)
             self.ASABApiService.initialize_web()
 
-        if "mqtt" in Config and self.DeploymentId:
+            mqtt_username = os.environ.get("MQTT_USERNAME")
+            mqtt_password = os.environ.get("MQTT_PASSWORD")
+
+        if "mqtt" in Config and self.DeploymentId and (mqtt_username or mqtt_password):
```

Comment:
You're still checking if mqtt is in the config. Which means users hav to add that section. We want the AOC to just work magically with no user engagement. So just remove this check from the if statement. That said, we should be loading the mqtt url as well...